src/main/resources/default/assets/styles/menu.scss

```diff
@@ -16,14 +16,31 @@ li.right-menu-icon {
 
 }
 
-@media (max-width: 768px) {
+@media (max-width: 991px) {
```

Comment:
These numbers and the ones below look sort of strange (off by one)?

Reply:
...will fit, otherwise the borders are missing at exactly 991px 🙈